drip/drips.py

```diff
@@ -21,24 +26,66 @@
 import logging
 
 
-def configured_message_classes() -> dict:
-    """[summary]
+def configured_message_classes() -> ChainMap:
+    """
+    Returns a ChainMap (basically a dict), between default settings and
+    settings.DRIP_MESSAGE_CLASSES which should be a dictionary of the form:
+
+        { message_class_name: class_path }
+
+    For example, if you define this at DRIP_MESSAGE_CLASSES:
+
+        {
+            'default': 'my.path.to.Drip',
+            'my_other_class': 'my.other.path.to.Drip',
+        }
+
+    The returning value of this method would be:
+
+        {
+            'default': 'my.path.to.Drip',
+            'my_other_class': 'my.other.path.to.Drip',
+        }
+
+    But, if you define this:
 
-    :return: [description]
-    :rtype: dict
+        {
+            'my_other_class': 'my.other.path.to.Drip',
+        }
+
+    The returning value of this method would be:
+
+        {
+            'default': 'drip.drips.Drip',
+            'my_other_class': 'my.other.path.to.Drip',
+        }
+
+    A 'default' key is added when it's not present on DRIP_MESSAGE_CLASSES.
     """
-    conf_dict = getattr(settings, 'DRIP_MESSAGE_CLASSES', {})
-    if 'default' not in conf_dict:
-        conf_dict['default'] = 'drip.drips.DripMessage'
-    return conf_dict
+    default_config = {'default': 'drip.drips.DripMessage'}
```

Comment:
Maybe add this as a constant outside of the function in order to make it more visible in the code

Reply:
Make a lot of sense, thanks Nico!